test-lit/semantics/getdefinition03-a.f90

```diff
@@ -0,0 +1,15 @@
+! Tests -fget-definition with INCLUDE
```

Comment:
I second @tskeith's request to separate these manual changes out from the scripted ones. The code review required for each is very different. For manual changes, we'll be looking at the changes to the test files line by line. For the scripted ones, most of the review will be on the script logic, with ported tests there simply to aid the review on the script. I would not look at each ported test line by line, or at least not very carefully. By combining changes like this together into one big patch you are forcing me to either check each ported one line by line, which is overkill and/or increasing the risk that I would not review the manual changes.